### ISSUE TITLE
Event not found in the WHISPers database

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispers#541]
There are events that I can see in the tabular data on the home page but then when I click on them from the home page to get to the event details page i get this: "Event 170704 not found in the WHISPers database". Seems most problematic when I'm logged in as a partner admin (not NWHC as org) and in Firefox but also problematic when logged in as myself (WHISPers admin) and sometimes doesn't work in Chrome either. Usually, once it starts doing this, it will do it for all events but sometimes it's more spotty.

**Comment by JChipault** [comment]
This is still happening pretty frequently in v13. I have to switch between firefox and chrome to try to get event details to display or just shut down the website completely and open it back up. Sometimes I just wait a few hours before using the website again and hope that does the trick.

**Comment by BlakeDraper** [comment]
@JChipault Are you seeing this on 'live' beta, test beta, or both? And which user(s) are you using to do it? I will need those details to recreate this behavior. 

The only way this occurs is if indeed the event does not exist in the database. The live and the test are two completely different databases, but if you are clicking in the table, it should only ever seek that event from the same source the tabular results came from. So this one has me confused. Is this a widespread reported issue? I only ask because I wonder if something is funky with your browser configuration.

**Comment by JChipault** [comment]
It's happened to Neil as well. And I think to one of the epis here. It seems random. Like I can see an event, then I click back home, then I try to get back to that event and am told that it's not found in the database. Then I just copy and paste the url from firefox to chrome and sometimes the event will load then in chrome. I've had it happen under multiple different logins, in both test and live, and in both firefox and chrome. I know the event exists in the database but I can't always get to it. I talked with Neil and we'll try to document the exact clicks that lead to this situation but it's a bit hard because it seems that this issue randomly comes and goes.

**Comment by BlakeDraper** [comment]
I am keeping my eyes open for this issue. I spent a while clicking around back and forth on the beta site today and still could not get this to happen. It has to be a failed look-up from the server because the "not found" message only appears for the user if the server lookup returns a "not found" error message. Not helpful to you, I know. I will stay vigilant about this for the moment and check with Aaron to see if he has any ideas. Let me know if you are able to gather more details on this at any point.

**Comment by JChipault** [comment]
Happened this morning. In firefox. I was playing around in the test site for a while. Then I opened the live site. I noticed that the live site logged me in automatically. I toggled back to the test site and was still logged in there. Toggled back to the live site and was still logged in as JChipault. Then I went to the home page on the live site and clicked on the event at the top of the list in the table, then i altered that event details url to try to jump to 200037 because I need to quality check 200037 and changing the url is currently the best way to jump to a specific event. Buuuuut got this screen below. When I copied and pasted the url into a chrome tab, it opened up to the event details.  
![image](https://user-images.githubusercontent.com/49287076/58336214-f3679200-7e08-11e9-83e5-53c7f4c7c4f5.png)

**Comment by JChipault** [comment]
And then I went back to the home page on firefox and clicked again on the top event on the home page table and now it won't let me open those details either. I had just seen these details before I jumped to 200037 url in Firefox but now 200076 won't show me details in firefox either.   Same for any other event I click on from the home page table; get "not found in WHISPers database". However, I then closed the tab in firefox completely and then open it back up, and could then see all the event details easily. 
![image](https://user-images.githubusercontent.com/49287076/58336457-5f49fa80-7e09-11e9-9fe0-29fa521af891.png)
